app/templates/components/ui-table/cell/events/view/sessions/cell-session-title.hbs

```diff
@@ -0,0 +1,14 @@
+{{record.title}}<br>
```

Comment:
We generally avoid the use of the br tag. You can use a hidden divider instead. 
Also there's no point of using a property name if you are using record.title simply to get the title.
May be you can use (get record column.propertyName) for it.

Reply:
@ritikamotwani when I use `<div class="hidden ui divider"></div>` instead of `<br>` the separation becomes too large.
_With hidden divider_
![Screenshot_2019-05-22 Session Sessions Apk19 Events Open Event(1)](https://user-images.githubusercontent.com/25428397/58174786-4da70c80-7cbc-11e9-8a76-8f1b322ca735.png)

_With br tag_
![Screenshot_2019-05-22 Session Sessions Apk19 Events Open Event](https://user-images.githubusercontent.com/25428397/58174789-4e3fa300-7cbc-11e9-9d3b-d35268049529.png)

Reply:
@shreyanshdwivedi , use the hidden divider one.